### DISCUSSION TITLE
evernote-backup not recognized after install

### DISCUSSION THREAD

**Discussion by aeerdogan** [vzhd1701/evernote-backup#67] [Q&A]
Hello, I installed with the "pipx" method for windows 11. Installation was successful, but the `evernote-backup init-db` command returns PATH error:
```
evernote-backup: The term 'evernote-backup' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```
Do I need to add this to PATH?

Thanks a lot.

**Comment by aeerdogan** [comment]
Ok l ran a `pipx list` command and i told me about my apps:
`apps are exposed on your $PATH at C:\Users\myusername\.local\bin`
So I added that to my PATH.  This time evernote-backup command ran. But it gave me some `No module named 'pyutil'` error, I guess that's a dependency, off to fix that now.

**Reply by goatroper** [reply]
I'm having the same issue trying to run 'evernote-backup'.  
on fresh py install, using pipx to do the app install.  Returns:
ModuleNotFoundError: No module named 'pyutil.version_class'
ModuleNotFoundError: No module named 'distutils'
Discovered from another blog that distutils was removed from Python 3.12.
tried last ver of py, 3.11, but still fails.

**Reply by rangermcfadden** [reply]
I faced this same problem. To fix it I **installed** pyutil 

`pip install pyutil`

then I **copied** the pyutil folder from:
`C:\Users\user\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\pyutil`

in to this one: `C:\Users\user\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\Local\pipx\pipx\venvs\evernote-backup\Lib\site-packages\`

**Comment by aeerdogan** [comment]
Got so many dependency issues. Uninstalled via pipx. 
Downloaded the exe, extracted in a folder, ran powershell there. I want to connect via oauth but it gives me `Error: No such option: --oauth` error.
